### ISSUE TITLE
Open Issues

### ISSUE THREAD

**Issue by sven-7** [ScudLee/anime-lists#360]
@ScudLee @sixtenbe @slohj  --- I know you guys are active every once and a while to push things thru. If willing to give me write access, I'm happy to go thru and clean up all the open issues and push what pull requests are still relevant thru.

**Comment by sven-7** [comment]
Bump to @ScudLee @sixtenbe @slohj

**Comment by sven-7** [comment]
Just checking again...

**Comment by reconman** [comment]
I think a switch to a fork is necessary, one owned by a Github organization. The main problem is that for personal repositories, only the owner can give you Collaborator status, which is ScudLee in this case. Other collaborators can't do anything.

In the case of organizations, multiple people can have a role similar to a repo owner. The Admin role can add new collaborators. That way, the repo will be future-proof as long as at least 1 admin is periodically available.

Organizations can be created under https://github.com/organizations/plan. Then fork with the organization, invite collaborators and assign them roles.

Afterwards we just need to convince people to use and update the fork.

**Comment by sven-7** [comment]
Okay. I think I'll take this to task. That way, it can get off @ScudLee's plate if he doesn't want to maintain anymore. Will look into this more today.

**Comment by sven-7** [comment]
@reconman yep - I've got something going. I'm going to try and close out the open issues and pull requests into the new one. Then maybe figure out how to pass it down.

**Comment by sven-7** [comment]
I'm talking about this here: https://github.com/ZeroQI/Absolute-Series-Scanner/issues/315

I've also tried reach out to @ScudLee on Kodi. 

Ultimately, I've set a new one up and if we work things out, it can just be merged into the master. Otherwise, we could move towards a new model.

Here is the temp: https://github.com/Anime-Lists/anime-lists

**Comment by reconman** [comment]
@sven-7 Have you noticed my PR in the new repo?

**Comment by sven-7** [comment]
@reconman I had not set up notifications just yet, so thanks for the heads up. Committed.

**Comment by reconman** [comment]
So, someone with write access copied over the commits from the new repository... Who was it?

**Comment by sven-7** [comment]
To follow up, that was @slohj 

Also - for anyone else, myself and whomever else wants to help, is maintaining a more up-to-date version here:  https://github.com/Anime-Lists/anime-lists
